code/modules/cargo/packs/emergency.dm

```diff
@@ -106,7 +106,7 @@
 
 /datum/supply_pack/emergency/radiatione_emergency
 	name = "Emergency Radiation Protection Crate"
-	desc = "Survive the Nuclear Apocalypse and Supermatter Engine alike with two sets of Radiation suits. Each set contains a helmet, suit, and Geiger counter. We'll even throw in a few pill bottles that are able to handles radiation and the affects of the poisoning."
+	desc = "Survive the Nuclear Apocalypse and Supermatter Engine alike with two sets of Radiation suits. Each set contains a helmet, suit and Geiger counter. We'll even throw in a few pill bottles that are able to handles radiation and the affects of the poisoning." //Skyrat change, fixed grammatical error
```

Comment:
"...helmet, suit, and Geiger counter."
vs.
"...helmet, suit and Geiger counter."
this is actually an incorrect change.
Look up the oxford comma.